### PR TITLE
Update relations

### PR DESCRIPTION
**Pull request by Vegter**
Model relations per source entity
Include volgnummer

### CODE REVIEW

File: gobcore/sources/relations.md [hunk 1]
```diff
@@ -107,18 +109,155 @@ To get the complete data of an entity it has to be joined with its relations.
...
+        volgnummer: 1,
+        begin_geldigheid: 1-1-2000,
+        eind_geldigheid: 1-1-2009
+    }
+	
+
+GOB-Relations:
+
+    {
+        _id: 'bag.adressen.ligt_in_wijk'
```
Comment (jasper-jager) [inline gobcore/sources/relations.md]: The value of id is missing in this example. According to the specs above it should be:

catalog.collection.id.attribute, eg bag.adressen.1.ligt_in_wijk

File: gobcore/sources/relations.md [hunk 2]
```diff
@@ -107,18 +109,155 @@ To get the complete data of an entity it has to be joined with its relations.
...
+        dst_collection: 'wijken'
+        dst_id: 1
+        volgnummer: int(1-1-2001)
+        registratiedatum: now
+        begin_geldigheid: 1-1-2001
+        eind_geldigheid: 1-1-2003
+    }
+
+    {
+        _id: 'bag.adressen.ligt_in_wijk'
```
Comment (jasper-jager) [inline gobcore/sources/relations.md]: Same as above

File: gobcore/sources/relations.md [hunk 3]
```diff
@@ -107,18 +109,155 @@ To get the complete data of an entity it has to be joined with its relations.
...
+        dst_collection: 'wijken'
+        dst_id: 2
+        volgnummer: int(1-1-2003)
+        registratiedatum: now
+        begin_geldigheid: 1-1-2003
+        eind_geldigheid: 1-1-2006
+    }
+
+    {
+        _id: 'bag.adressen.ligt_in_wijk'
```
Comment (jasper-jager) [inline gobcore/sources/relations.md]: Same as above

File: gobcore/sources/relations.md [hunk 4]
```diff
@@ -107,18 +109,155 @@ To get the complete data of an entity it has to be joined with its relations.
...
+        dst_collection: 'wijken'
+        dst_id: 1
+        volgnummer: int(1-1-2006)
+        registratiedatum: now
+        begin_geldigheid: 1-1-2006
+        eind_geldigheid: 1-1-2007
+    }
+
+    {
+        _id: 'bag.adressen.ligt_in_wijk'
```
Comment (jasper-jager) [inline gobcore/sources/relations.md]: Same as above

File: gobcore/sources/relations.md [hunk 5]
```diff
@@ -107,18 +109,155 @@ To get the complete data of an entity it has to be joined with its relations.
...
+        dst_collection: 'wijken'
+        dst_id: 3
+        volgnummer: int(1-1-2007)
+        registratiedatum: now
+        begin_geldigheid: 1-1-2007
+        eind_geldigheid: 1-1-2009
+    }
+
+    {
+        _id: 'bag.adressen.ligt_in_wijk'
```
Comment (jasper-jager) [inline gobcore/sources/relations.md]: Same as above

**Comment by Robkromwijk** [comment]
GOB relations should also cope with manyreferences

**Pull request merged**